src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -112,11 +115,28 @@ public async Task<Result<int>> Handle(CreateInvitationCommand request, Cancellat
             try
             {
                 invitation.MeetingId = await CreateOutlookMeeting(request, meetingParticipants, invitation, project.Name);
+            }
+            catch
+            {
+                try
+                {
+                    _logger.LogWarning($"Trying to use fallback solution for creating outlook meeting since meeting API failed for user with oid {_currentUserProvider.GetCurrentUserOid()} and ivitation id {invitation.Id}.");
```

Comment:
move codeline 123 + 124 to line 121, just before try. The catch on line 127 should catch only if _smtpService.SendSmtpWithInviteAsync fails